app/src/main/graphql/project.graphql

```diff
@@ -46,29 +46,30 @@ mutation CreateComment($body: String!, $commentableId: ID!, $parentId: ID, $clie
   }
 }
 
-query GetProjectComments($slug: String!, $limit: Int!, $cursor: String){
-      project(slug: $slug) {
-        collaborators {
-          edges {
-            node {
-              id
-              name
-            }
-          }
+query GetProjectComments($slug: String!, $limit: Int!, $cursor: String) {
+  project(slug: $slug) {
+    id
+    collaborators {
+      edges {
+        node {
+          id
+          name
         }
-        comments(first: $limit, after: $cursor) {
-          edges {
-            cursor
-            node {
-              authorBadges
-              ...comment
-            }
-          }
-          pageInfo {
-                  ...pageInfo
-           }
-          totalCount
+      }
+    }
+    comments(first: $limit, after: $cursor) {
+      edges {
+        cursor
+        node {
+          authorBadges
```

Comment:
Same here, if authorBadges is included in the comment fragment

Reply:
👍🏻